test/publisher/index.ts

```diff
@@ -223,7 +192,15 @@ describe('Publisher', () => {
       opentelemetry.trace.setGlobalTracerProvider(provider);
 
       tracingPublisher.publish(buffer);
-      assert.ok(exporter.getFinishedSpans());
+      const spans = exporter.getFinishedSpans();
```

Comment:
Nice this looks better

Reply:
yes, the problem is that it wasn't checking if it had a span so it always was successful (even now in master) if I change this to check the length >= 1 it will fail. 

Only I am not well versed in this library or its tests to see were the issue is. It does like that _constructSpan always exits early due the first if-statement.

Reply:
So its still not creating a span?

Reply:
Yeah, I tried it a minute ago and it looks like it's still not getting any spans. I thought maybe it's because you have to add a tracer, but that doesn't seem to have changed it. I don't know much about OT either :)

Slightly unrelated: `assert.equal` shows up as deprecated for me, and it suggests `assert.strictEqual`.

Reply:
I'll take a look

Reply:
@weyert I'll open a PR to your fork. I think my original suggestions were too lofty, I want to sync up with the pubsub folks before changing things too much.